cmd/internal/planetscale_edge_mysql.go

```diff
@@ -211,6 +211,13 @@ func getJsonSchemaType(mysqlType string) StreamProperty {
 		return StreamProperty{Types: []string{"null", "string"}, CustomFormat: "timestamp_with_timezone"}
 	}
 
+	if strings.HasSuffix(mysqlType, "int") {
```

Comment:
Can you rely on this format for all integer types? Thinking of the Go convention of like `int32` vs strings like `bigint`.

Reply:
Yeah, `bigint` is captured above. 

From Mysql's list here : https://dev.mysql.com/doc/refman/8.0/en/integer-types.html

Reply:
I'll do another pass on the data types and make sure we do the right thing for Stitch here.